shared/clinical-code-sets/referrals/pain-management/1/pain-management.ctv3.txt

```diff
@@ -0,0 +1,11 @@
+XaAbA	Referral by pain management nurse
+XaAbA	Referral from pain management nurse
+Xaacg	Private referral to pain management service
+XaAcM	Referral to pain management service
+XaAfZ	Referral to pain management specialist
+XaAh2	Referral to pain management nurse
+XaAZh	Referral from pain management specialist
+XaAZh	Referral by pain management specialist
+XaB4p	Referral to acute pain service
+XaFs5	Referral to rapid access chest pain clinic
```

Comment:
I wonder whether this should be here. i suspect rapid access chest pain clinic would be about diagnosing cardiovascular conditions, rather than for managing the pain.